### ISSUE TITLE
are they any exemple for  simple sniper bot 

### ISSUE THREAD

**Issue by ghost** [flashbots/simple-arbitrage#32]
that use flashbot ?

**Comment by henrykash** [comment]
yes follow this guy has a public repo with a sniper bot implementation  www.github.com/henrykash

**Comment by ghost** [comment]
> yes follow this guy has a public repo with a sniper bot implementation [www.github.com/henrykash](http://www.github.com/henrykash)

he dont use flashbot to send bundle

**Comment by henrykash** [comment]
if you want you can just use the flashbot  RPC to send a bundle ,  I was responding in response to your question about an example of a sniper bot

**Comment by ghost** [comment]
> if you want you can just use the flashbot RPC to send a bundle , I was responding in response to your question about an example of a sniper bot

i asked about one who use flashbot to get priority included by bribing
